dist/aurora.js

```diff
@@ -18,13 +18,18 @@ function getAngularImagePriorityCount() {
 }
 
 // Detects Vue version for Nuxt apps
-// Nuxt doesn't expose its own version yet, but does expose its Vue version
+// Nuxt only exposes its own version for Nuxt apps on v3.3+
 function getVueVersionForNuxt() {
   const nuxt2Version = window.$nuxt?.$root?.constructor?.version;
   const nuxt3Version = document.querySelector('#__nuxt')?.__vue_app__?.version;
   return nuxt2Version || nuxt3Version;
 }
 
+// Detects Nuxt version directly for Nuxt 3.3 and later
+function getNuxtVersion() {
+  return __unctx__?.get('nuxt-app')?.use()?.versions.nuxt;
```

Comment:
Can't hurt

```suggestion
  return __unctx__?.get('nuxt-app')?.use()?.versions?.nuxt;
```

Reply:
Makes sense! Thanks for making the change!